docs/observe/observing-a-service.md

```diff
@@ -0,0 +1,90 @@
+# Observing a Service with Prometheus
+
+There are mainly two systems involved in collecting and visualizing the metrics. [Prometheus](https://prometheus.io/) is used to collect the metrics from the Ballerina service and [Grafana](https://grafana.com/) can connect to Prometheus and visualize the metrics in the dashboard.
+
+To understand how you can observe Ballerina services, let’s consider a service that converts JSON to XML.
+
+![alt text](../../assets/img/prometheus-grafana.png)
+
+As inferred by the above image, we will be getting metrics using Prometheus and displaying it using Grafana.
+
+## Set up the project 
+
+1. Open VS Code.
+   > **Tip**: Download and install [VS Code](https://code.visualstudio.com/Download) if you do not have it already. Find the extension for Ballerina in the VS Code marketplace if you do not have it. For instructions on installing and using it, see [The Visual Studio Code Extension](https://ballerina.io/learn/tools-ides/vscode-plugin/).
```

Comment:
Can we add the exact name of the Ballerina extension which is "Ballerina" and make sure the version compatibility of B7A plugin and VSCode plugin.

Reply:
Done.